doc/src/modules/geometry/index.rst

```diff
@@ -18,10 +18,10 @@ Available Entities
 
 The following entities are currently available in the geometry module:
 
-* ``Point``
-* ``Line``, ``Ray``, ``Segment``
-* ``Ellipse``, ``Circle``
-* ``Polygon``, ``RegularPolygon``, ``Triangle``
+* :class:`~sympy.geometry.point.Point`
+* :class:`~sympy.geometry.line.Segment`, :class:`~sympy.geometry.line.Ray`, :class:`~sympy.geometry.line.Segment`
```

Comment:
Wait a minute, the first one has to be `Line`, not `Segment`.

I missed that in the first view. Otherwise LGTM.